lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_cleaning.erb

```diff
@@ -11,6 +11,10 @@ To minimise the risk of the virus spreading you should:
 - provide more bins and empty them more often
 - clear workspaces and remove waste and belongings from the area at the end of a shift
 - use different gowns and towels for each client
+- make sure that housekeeping staff follow government handwashing guidelines, and make a checklist of all surfaces to be cleaned when each guest vacates
```

Comment:
Should we link to the handwashing guidance?

Reply:
not sure that there is a link specifically for that?

Reply:
I think it's the wash your hands for 20 seconds, thoroughly etc.

Reply:
No worries - if there''s not a clear link to add now we can, look at this in the future if its a problem